solidity/test/LiquidityProtection.js

```diff
@@ -98,7 +98,7 @@ describe('LiquidityProtection', () => {
                     await baseToken.approve(converter.address, RESERVE1_AMOUNT);
                 }
 
-                await converter.addLiquidity(
+                await converter.methods['addLiquidity(address[],uint256[],uint256)'](
```

Comment:
we can delete all the "new" non array function overloads and clean this up

Reply:
@barakman in any case, assuming that you'd still have to do it - DRY it. There is no reason to have all these selectors repeated all the time.